src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,366 @@
+
+Infine, se usati in un header di risposta HTTP, alcuni CDN trasformeranno automaticamente un "preload" in un [HTTP/2 push](#http2-push) che può sovraccaricare le risorse memorizzate nella cache.
+
+### `prefetch`
+
+L'hint [`prefetch`](https://web.dev/link-prefetch/) ci permette di avviare richieste a bassa priorità che ci aspettiamo vengano usate nella prossima navigazione. L'hint scaricherà le risorse e le rilascerà nella cache HTTP per un utilizzo successivo. È importante notare che `prefetch` non eseguirà o elaborerà in altro modo la risorsa, e per eseguirlo la pagina dovrà comunque chiamare la risorsa tramite il tag `<script>`.
+
+```html
+<link rel="prefetch" as="script" href="next-page.bundle.js">
+```
+
+Esistono diversi modi per implementare la logica di previsione di una risorsa, che potrebbe essere basata su segnali come il movimento del mouse dell'utente, flussi/percorsi comuni degli utenti o anche sulla base di una combinazione di entrambi in aggiunta al machine learning.
+
+Fai attenzione, a seconda della [qualità](https://github.com/andydavies/http2-prioritization-issues#current-status) della prioritizzazione HTTP/2 del CDN utilizzato, la prioritizzazione di `prefetch` potrebbe migliorare le prestazioni o renderle più lente, dando priorità alle richieste di `prefetch` e togliendo della larghezza di banda importante per il caricamento iniziale. Assicurati di ricontrollare la CDN che stai utilizzando e adattati per prendere in considerazione alcune delle migliori pratiche condivise di [Andy Davies's](https://andydavies.me/blog/2020/07/08/rel-equals-prefetch-and-the-importance-of-effective-http-slash-2-prioritisation/).
+
+### Native lazy loading
+
+L'hint [native lazy loading](https://web.dev/browser-level-image-lazy-loading/) è un'API del browser nativa per differire il caricamento di immagini offscreen e iframe. Usandolo, le risorse che non sono necessarie durante il caricamento iniziale della pagina non avvieranno una richiesta di rete, questo ridurrà il consumo di dati e migliorerà le prestazioni della pagina.
+
+```html
+<img src="image.png" loading="lazy" alt="…" width="200" height="200">
+```
+
+Tieni presente che l'implementazione di Chromium della logica delle soglie di lazy-loading è stata storicamente piuttosto [conservativa](https://web.dev/browser-level-image-lazy-loading/#distance-from-viewport-thresholds), mantenendo il limite offscreen a 3000px. Durante l'ultimo anno il limite è stato attivamente testato e migliorato per allineare meglio le aspettative degli sviluppatori e, infine, spostare le soglie a 1250px. Inoltre, non esiste [nessuno standard nei browser](https://github.com/whatwg/html/issues/5408) e non è ancora possibile per gli sviluppatori web ignorare le soglie predefinite fornite dai browser.
+
+## Resource hints
+
+Sulla base dell'archivio HTTP, passiamo all'analisi delle tendenze del 2020 e confrontiamo i dati con il precedente dataset del 2019.
+
+### Hints adoption
```

Comment:
@chefleo did you see this? Hidden under a "Load more..." so you might have missed it.

Reply:
`Hints adoption` 🤔 So we are keeping the word `Hints` not translated to make it easier to people to connect the two concept Hints <-> Resource Hints. If we translate `adoption` it will be `adozione` as the word `uso` or `usare` is a more common word for adoption. @giopunt @chefleo thoughts?